pkg/acsengine/engine.go

```diff
@@ -283,6 +284,7 @@ func prepareTemplateFiles(properties *api.Properties) ([]string, string, error)
 	var files []string
 	var baseFile string
 	if properties.OrchestratorProfile.OrchestratorType == api.DCOS188 ||
+		properties.OrchestratorProfile.OrchestratorType == api.DCOS190 ||
```

Comment:
@amanohar Since 1.9 isn't GA yet, I was thinking of letting 1.8.8 as default version so that PR can still be merged, and making another PR once 1.9 hit GA since the bootstrap url will also change from `EarlyAccess` to `Stable` and the package ids will be differents.  
Let me know what you think.